go-controller/hack/lint.sh

```diff
@@ -1,18 +1,28 @@
 #!/usr/bin/env bash
 VERSION=v1.64.8
-extra_flags=""
+: "${GOLANGCI_LINT_VERSION:=$VERSION}"
+extra_flags=(--verbose --print-resources-usage --modules-download-mode=vendor --timeout=15m0s)
```

Comment:
What is the reason behind adding these extra flags?

Reply:
just do de-dup them in the two below golangci-lint commands